src/ecalc/libraries/neqsim/tests/integration_tests/test_not_threadsafe.py

```diff
@@ -14,16 +15,17 @@
 
 def thread_job(has_water: bool):
     neqsim = java_gateway.jvm.neqsim
-    Fluid = neqsim.thermo.Fluid
+    Fluid = neqsim.thermo.Fluid()
 
     assert Fluid.isHasWater() is not has_water
     Fluid.setHasWater(has_water)
 
 
+@pytest.mark.xfail(reason="No longer a static variable in NeqSim, could remove test if not new case found")
 def test_run_neqsim_in_2_threads():
     neqsim = java_gateway.jvm.neqsim
     # Initially, has water is false
-    Fluid = neqsim.thermo.Fluid
+    Fluid = neqsim.thermo.Fluid()
```

Comment:
isHasFluid is no longer a static parameter, and can't be accessed directly

Reply:
can be replaced with another static variable or static set/get method, if there is any. it was used as a simple example demonstrating that 2 threads would lead to race conditions, to avoid anyone in future to remove the lock in the code, which would in 99% of the cases silently deliver incorrect results and only sometimes fail...